pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,321 @@ excerpt:
+Device: ca01h/51713d    Inode: 100671105   Links: 1
+Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
+Context: unconfined_u:object_r:var_lib_t:s0
+Access: 2017-12-27 12:35:58.643278377 +0000
+Modify: 2017-12-27 12:35:58.643278377 +0000
+Change: 2017-12-27 12:36:13.019162417 +0000
+Birth: -
+```
+
+- The configuration parameters `ca_certificate_path`, `ca_certificate_key_path` and `ca_certificate_chain_path` are specified in the DC/OS configuration file `$DCOS_INSTALL_DIR/genconf/config.yaml` and point to the relevant locations in the file system. Example (commands issued on the bootstrap node):
+
+```bash
+cd $DCOS_INSTALL_DIR
+cat genconf/config.yaml
+```
+```bash
+[...]
+ca_certificate_path: genconf/dcos-ca-certificate.crt
+ca_certificate_key_path: genconf/dcos-ca-certificate-key.key
+ca_certificate_chain_path: genconf/dcos-ca-certificate-chain.crt
+[...]
+```
+Note that `ca_certificate_chain_path` is an optional parameter when setting up DC/OS Enterprise with a custom CA certificate when the latter is a self-signed root certificate.
+
+## Installation
+Proceed with the installation as described in the [documentation of the Advanced Installer](/1.10/installing/ent/custom/advanced/#install-dcos).
+
+## Verify installation
+One method of verifying that the DC/OS was installed properly with the custom CA certificate is to initiate a TLS connection to the Admin Router which, after installation, will have a certificate signed by the custom CA. 
+
+In order to do that you need first to obtain the DC/OS CA bundle of the deployed cluster. [This page](/1.10/security/ent/tls-ssl/get-cert/) shows how you can do that.
+
+Provided you have obtained the DC/OS bundle and stored it in a file named `dcos-ca.crt`, issue the following command in the directory containing the `dcos-ca.crt` file in order to check that the Admin Router on a master node uses a certificate signed with the custom CA certificate:
+
+```bash
+openssl s_client -verify_ip <private_ip_master_nodeX> -CAfile dcos-ca.crt -connect <public_ip_master_nodeX>:443 | grep -e "s:" -e "i:" -e "return code:"
+```
+
+The output should look similar to the following:
+
+```bash
+depth=3 DC = io, DC = integration-test, C = DE, ST = Utopia, O = DC/OS, OU = Brogrammer Unit, CN = Integration Test Root CA
+verify return:1
+depth=2 DC = io, DC = integration-test, C = DE, ST = Utopia, O = DC/OS, OU = Brogrammer Unit, CN = Integration Test Intermediate CA 01
+verify return:1
+depth=1 DC = io, DC = integration-test, C = DE, ST = Utopia, O = DC/OS, OU = Brogrammer Unit, CN = Integration Test Intermediate CA 02
+verify return:1
+depth=0 C = US, ST = CA, L = San Francisco, O = "Mesosphere, Inc.", CN = AdminRouter on 172.31.12.45
+verify return:1
+ 0 s:/C=US/ST=CA/L=San Francisco/O=Mesosphere, Inc./CN=AdminRouter on 172.31.12.45
+   i:/DC=io/DC=integration-test/C=DE/ST=Utopia/O=DC/OS/OU=Brogrammer Unit/CN=Integration Test Intermediate CA 02
```

Comment:
`Brogrammer Unit` made it our public documentation 🎉 😆

Reply:
I'm not sure that's a good thing. The idea of a Brogrammer Unit might be seen to reinforce some unfortunate stereotypes. I'm sure we could have an equally amusing name that's less open to misinterpretation.

Reply:
Thanks for raising that point, Jon. I agree. Internally it can be considered amusing, but to the public this is an unnecessary attack surface.

As one of the last polishing steps this should be replaced with a different word. @jongiddy do you volunteer for finding the equally amusing name? :)

@ahairape please own that point.

Please also make sure that we're not publishing the corresponding files.

Reply:
Schlaraffenland? Given that it exists in the state of Utopia

Reply:
I think we could may be use Schlaraffenland as a value in the **C** or **ST** fields of the certificate subject, but it does not make sense to me in the **OU** field. I'll change it to a prosaic _Programmer Unit_ for now, we can change it to something more amusing later.